camus-sweeper/src/main/java/com/linkedin/camus/sweeper/utils/RelaxedAvroKeyOutputFormat.java

```diff
@@ -43,4 +50,12 @@
 
   }
 
+  @Override
+  public synchronized OutputCommitter getOutputCommitter(TaskAttemptContext context) throws IOException {
+    if (this.commiter == null) {
```

Comment:
Lazy initialization. This is the same as the implementation in the parent method (`FileOutputFormat.getOutputCommitter`).